recipes/default.rb

```diff
@@ -29,7 +32,7 @@
 
 directory node.loggly.tls.cert_path do
   owner 'root'
-  group 'syslog'
+  group node.rsyslog.priv_group || 'root'
```

Comment:
If a group of nil is specified, chef may just default to root... if that's the case, I can just default to what comes from rsyslog cookbook.  I will test it out.  However, this is a little more explicit and won't break if Chef ever changes the behaviour for a nil group value.